product_variant_change_attribute_value/tests/test_product_variant_change_attribute_value.py

```diff
@@ -88,9 +89,10 @@ def test_fields(self):
     def test_actions_field_filter(self):
         wiz = self._get_wiz()
         self.assertEqual(len(wiz.attributes_action_ids), len(wiz.attribute_value_ids))
-        wiz.filter_attribute_id = self.legs
+        with Form(wiz) as res:
+            res.filter_attribute_id = self.legs
         self.assertEqual(
-            len(wiz.attributes_action_ids),
+            len(res.attributes_action_ids),
```

Comment:
Should the assert not be in the context manager ?

Reply:
Maybe. Probably. I have to switch on something else. I'll come back to this later :smile: